external/sql/README.md

```diff
@@ -0,0 +1,101 @@
+# Storm SQL
+
+Compile SQL queries to Storm topologies.
+
+## Usage
+
+Run the ``storm sql`` command to compile SQL statements into Trident topology, and submit it to the Storm cluster
+
+```
+$ bin/storm sql <topo-name> <sql-file>
```

Comment:
minor: can you have `topo-name` as the last argument to be in sync with the `storm jar` command where topology name is the last argument?